x-pack/metricbeat/module/panos/system/_meta/fields.yml

```diff
@@ -0,0 +1,114 @@
+- name: system
+  type: group
+  release: beta
+  description: >
+    system
+  fields:
+    - name: uptime.days
+      type: integer
+      description: >
+        Uptime in days
+    - name: uptime.hours
+      type: keyword
```

Comment:
is a keyword or integer?

Reply:
This value was in format h:m, so I broke it up and now have uptime.hours and uptime.minutes